akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/EventSourcedBehavior.scala

```diff
@@ -208,6 +208,10 @@ object EventSourcedBehavior {
    * Specifically BackOff to prevent resume being used. Resume is not allowed as
    * it will be unknown if the event has been persisted.
    *
+   * This supervision is only around the event sourced behavior not any outer setup/withTimers
+   * block. If using restart any actions e.g. scheduling timers, can be done on the PreRestart
+   * signal or on the RecoveryCompleted signal.
```

Comment:
I don't understand that sentence. Do you mean with "using restart any actions"?

Reply:
I read it as "if using restart"  = if the backoff restart supervision is enabled by defining it with this method

Reply:
In that case, a comma could help. 

```suggestion
   * block. If using restart, any actions e.g. scheduling timers, can be done on the PreRestart
```